### ISSUE TITLE
Wrong documented return type from verify/2

### ISSUE THREAD

**Issue by norpan** [princemaple/elixir-jwt#2]
```
@spec verify(binary, map) :: {:ok, map} | {:error, binary}
```

When a claim fails, it does not return binary, but a keyword list.

```
iex(1)> JWT.verify(JWT.sign(%{foo: "bar", exp: :os.system_time(:seconds) - 1}, key), key)
{:error, [exp: 1548504635]}
```

**Comment by princemaple** [comment]
You are right

**Comment by princemaple** [comment]
fixed in https://github.com/princemaple/elixir-jwt/commit/82332aa962a2d5f5ecbe757948dd91c7137e6f21

**Comment by princemaple** [comment]
released 1.1.1
